Source/WebKit/WebProcess/WebProcess.cpp

```diff
@@ -960,6 +964,78 @@ void WebProcess::didReceiveMessage(IPC::Connection& connection, IPC::Decoder& de
     LOG_ERROR("Unhandled web process message '%s' (destination: %" PRIu64 " pid: %d)", description(decoder.messageName()), decoder.destinationID(), static_cast<int>(getCurrentProcessID()));
 }
 
+void WebProcess::setRemoteOpener(WebCore::GlobalWindowIdentifier&& remoteWindowIdentifier, WebCore::GlobalWindowIdentifier&& openerWindowIdentifier)
+{
+    auto it = WebCore::AbstractDOMWindow::allWindows().find(remoteWindowIdentifier);
```

Comment:
Seems we could use `get()`. Let's not use iterators unless there is a performance benefit (avoid double hash lookup). Same comment applies below.